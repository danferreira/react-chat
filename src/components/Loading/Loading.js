import React from 'react';

import './Loading.css';

const Loading = () =>
    <div className="center">
        <div className="spinner"></div>
    </div>

export default Loading;