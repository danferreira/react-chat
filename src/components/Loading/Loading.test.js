import React from 'react';
import { shallow } from 'enzyme';

import Loading from './Loading';

describe('<Loading />', () => {
    it('should render properly', () => {
        const wrapper = shallow(<Loading />);

        expect(wrapper).toMatchSnapshot();
    })
});