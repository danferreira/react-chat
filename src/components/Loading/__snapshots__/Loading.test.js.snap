// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should render properly 1`] = `
<div
  className="center"
>
  <div
    className="spinner"
  />
</div>
`;
